whitelabelling_bar/package.json

```diff
@@ -0,0 +1,130 @@
+{
+    "name": "whitelabelling_bar",
+    "version": "1.0.5",
+    "description": "A dashboard plugin that adds a bar at the bottom of the dashboard for white-labelling content.",
+    "author": "GoodData Corporation",
+    "private": true,
+    "license": "MIT",
```

Comment:
@dremileta in the public repo, please always fix the license path and set private property to false